Makefile

```diff
@@ -1,8 +1,15 @@
 CFLAGS += -Wall -Wextra
+VERSION = r$(shell git rev-list --count HEAD)-$(shell git rev-parse --short=8 HEAD)
 
 all: uradvd
 uradvd: uradvd.o
 
+uradvd.o:
+	$(CC) $(CFLAGS) -DVERSION='"$(VERSION)"' -c uradvd.c
+
+version:
```

Comment:
Should be added to `.PHONY`

Reply:
resolved in 8256fd25aa6ba9625b0f0a184d0632353beadfaa